src/CommandingOfficers/Meridian.java

```diff
@@ -0,0 +1,375 @@
+      infoPages.add(new InfoPage(
+          "Passive:\n" + 
+          "- Tanks and artillery both cost 6500\n" +
+          "- Tanks and artillery may transform into each other at no cost"));
+      infoPages.add(new InfoPage(
+          "Change and Flow ("+ChangeAndFlow.COST+"):\n" +
+          "Gives an attack and defense boost of "+ChangeAndFlow.BASIC_BUFF+"%\n" +
+          "Any unit that has transformed this turn is refreshed."));
+      infoPages.add(new InfoPage(
+          "Vehicular Charge ("+VehicularCharge.COST+"):\n" +
+          "Gives an attack and defense boost of "+VehicularCharge.BASIC_BUFF+"%\n" +
+          "Refreshes all land vehicles, but refreshed units suffer an attack and defense penalty of "+POST_REFRESH_STAT_ADJUSTMENT+"%\n"));
+    }
+    @Override
+    public Commander create()
+    {
+      return new Meridian();
+    }
+  }
+
+  /** A list of all the units I've refreshed and need to nerf. */
+  private ArrayList<Unit> justTransformed = new ArrayList<Unit>();
+  private ArrayList<Unit> toBeNerfed = new ArrayList<Unit>();
+  private static final int POST_REFRESH_STAT_ADJUSTMENT = -25;
+
+  public Meridian()
+  {
+    super(coInfo);
+
+    // Meridian's tanks and arty cost the same
+    UnitModel tank = getUnitModel(UnitModel.UnitEnum.TANK);
+    UnitModel arty = getUnitModel(UnitModel.UnitEnum.ARTILLERY);
+    tank.moneyCostAdjustment      -= 500;
+    arty.moneyCostAdjustment      += 500;
+    tank.possibleActions.add(new Transform(this, arty));
+    arty.possibleActions.add(new Transform(this, tank));
+
+    addCommanderAbility(new ChangeAndFlow(this));
+    addCommanderAbility(new VehicularCharge(this));
+  }
+
+  @Override
+  public GameEventQueue initTurn(GameMap map)
+  {
+    justTransformed.clear();
+    toBeNerfed.clear();
+    modifyAbilityPower(40);
+    return super.initTurn(map);
+  }
+
+  /**
+   * Troops that have been refreshed by Meridian's bigger power get a stat nerf
+   */
+  @Override
+  public void applyCombatModifiers(BattleParams params, boolean amITheAttacker)
+  {
+    if( toBeNerfed.contains(params.attacker) )
+    {
+      params.attackFactor += POST_REFRESH_STAT_ADJUSTMENT;
+    }
+    if( toBeNerfed.contains(params.defender) )
+    {
+      params.defenseFactor += POST_REFRESH_STAT_ADJUSTMENT;
+    }
+  }
+
+  /**
+   * Change and Flow refreshes units that have transformed this turn
+   */
+  private static class ChangeAndFlow extends CommanderAbility
+  {
+    private static final String STRONGARM_NAME = "Change and Flow";
+    private static final int COST = 4;
+    private static final int BASIC_BUFF = 10;
+    
+    CODamageModifier damageMod = null;
+    CODefenseModifier defenseMod = null;
+    Meridian COcast;
+
+    ChangeAndFlow(Meridian commander)
+    {
+      super(commander, STRONGARM_NAME, COST);
+
+      damageMod = new CODamageModifier(BASIC_BUFF);
+      defenseMod = new CODefenseModifier(BASIC_BUFF);
```

Comment:
Since so many COs do the +10/10 thing, I'm tempted to make a modifier type that does both, just to make this less cumbersome. It will happen. Eventually. Probably.

Reply:
Or we can just keep copy-pasting that code over and over again...

Honestly, it might make sense to make a subclass of `CommanderAbility` to do that for us.